gravitee-apim-cypress/cypress/integration/rest-api/apis/jwt-policy-tests.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const jwt = require('jsonwebtoken');
+import { ADMIN_USER, API_PUBLISHER_USER } from 'fixtures/fakers/users/users';
+import { deleteApi, deployApi, getApiKeys, importCreateApi, startApi, stopApi } from 'commands/management/api-management-commands';
+import { publishPlan, closePlan } from 'commands/management/api-plan-management-commands';
+import { PlanSecurityType } from 'model/plan';
+import { ApiImportFakers } from 'fixtures/fakers/api-imports';
+import { ApiImport } from '@model/api-imports';
+import {
+  closeApplicationSubscription,
+  createApplication,
+  deleteApplication,
+  subscribeApplication,
+} from '@commands/management/application-management-commands';
+import { ApplicationFakers } from '@fakers/applications';
+import { requestGateway } from 'support/common/http.commands';
+import * as faker from 'faker';
+import { PolicyFakers } from '@fakers/policies';
+
+context('Create and test a JWT policy', () => {
```

Comment:
This test case checks 2 things :
- the behavior of the API key plan
- the behavior of the JWT policy

Maybe we should have 2 distinct test cases, 1 checking the API key plan, 1 other checking the JWT policy ?

Reply:
Ok, I'll create 2 separate scenarios.